string actionName
string[] argVals
string[] paramNames
string[] paramVals
---
int64 success_bool